### ISSUE TITLE
Is there a way to completely hide the data types section from the docs

### ISSUE THREAD

**Issue by watery** [stoicflame/enunciate#916]
I'd like to hide the `Data Types` documentation section, and leave types to be navigated from services: is this possible?

**Comment by stoicflame** [comment]
Not short of providing your own custom documentation template.

I'll track this as an enhancement request for now.

**Comment by watery** [comment]
Fine, thank you.

**Comment by stoicflame** [comment]
Fixed at 3898d8cae:

```xml
<enunciate>
  <modules>
    <docs includeDataTypesHomeIndex="false"/>
```

**Comment by stoicflame** [comment]
[Enunciate 2.12](https://github.com/stoicflame/enunciate/releases/tag/v2.12.0) has been released.
